package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/ably/ably-js/issues",
     "email": "support@ably.io"
   },
-  "main": "./nodejs/index.js",
+  "main": "./browser/static/ably-node.js",
```

Comment:
Why this change?
Also this will presumably break the react-native and browser builds below unless you change the main-script alias for them to match this change

Reply:
The reason for this to change was so that I could exclude the `nodejs` directory from the package entirely. It makes sense to me to have a clear distinction between 'source code' and 'build output' so this change means the entire `nodejs` directory is now just source code and paves the way for us to do something like a top level `src` and `dist` folder structure in future. You're right about me needing to change the aliases for rn and browser so I've force pushed that change now.